go-chaos/internal/manifests/worker.yaml

```diff
@@ -16,15 +16,15 @@ spec:
     spec:
       containers:
         - name: worker
-          image: gcr.io/zeebe-io/worker:REPLACE
+          image: gcr.io/zeebe-io/worker:{{ imageTag }}
```

Comment:
:thought_balloon: Because we parameterize two things now, I figured we can start using a more explicit notation than just `REPLACE`. If we parameterize 3 or more, we should look into using go template ;)

Reply:
Rule of three :smile: